src/Ninja/G1CP/Content/Misc/npc.d

```diff
@@ -219,3 +219,12 @@ func void G1CP_NpcIdSetAiVar(var int npcInstance, var string aiVarName, var int
     var C_Npc slf; slf = Hlp_GetNpc(npcInstance);
     G1CP_NpcSetAiVar(slf, aiVarName, value);
 };
+
+/*
+ * Change the value of an NPC's integer variable
+ */
+funt int G1CP_NpcChangeIntVar(var string npcName, var string npcProp, var int originalInt, var int correctedInt) {
```

Comment:
Type: funt -> func

Creating this extra layer of a function call, does not makes sense as long as there is no other fix of that same type.

Reply:
Thanks for the typo hint!

There is at least one other fix queued that will benefit from this refactor, so I think it's worth refactoring.